README.md

```diff
@@ -32,8 +32,15 @@ ProGuard
 
 If you are using ProGuard you might need to add the following options:
 ```
+-dontnote retrofit2.Platform
+-dontwarn retrofit2.Platform$Java8
 -dontwarn okio.**
 -dontwarn javax.annotation.**
+-keepattributes Signature
+-keepattributes Exceptions
+-keepclasseswithmembers class * {
```

Comment:
If I use these interfaces this block is needed. Maybe note about adding this block is needed here